server/views/pages/overview/partials/educationAndTrainingTab/_inductionShortQuestionSet.njk

```diff
@@ -53,11 +48,13 @@ as the follow up questions are different.
           Other training or qualifications
         </dt>
         <dd class="govuk-summary-list__value">
-          {# TODO: Populate values and handle the for each statement #}
           <ul class="govuk-list">
-            {# for each qualification recorded #}
-            <li>[Qualification value]</li>
-            {# endfor #}
+            {% for additionalTraining in educationAndTraining.data.shortQuestionSetAnswers.additionalTraining %}
```

Comment:
I think this `for` and `if` are slightly wrong. I think you want to do it like I've done it here (for the long question set - you can replace `longQuestionSet` with `shortQuestionSet` and it should work):
```
          <ul class="govuk-list">
            {% for additionalTraining in educationAndTraining.data.longQuestionSetAnswers.additionalTraining %}
              {% if additionalTraining == 'OTHER' %}
                <li>{{ educationAndTraining.data.longQuestionSetAnswers.otherAdditionalTraining }}</li>
              {% else %}
                <li>{{ additionalTraining | formatAdditionalTraining }}</li>
              {% endif %}  
            {% endfor %}
          </ul>
```

Reply:
Yeah that makes more sense!

Reply:
Fixed